video/analyze.js

```diff
@@ -235,9 +259,9 @@ function analyzeSafeSearch (gcsUri) {
     })
     .then((results) => {
       // Gets unsafe content
-      const safeSearchResults = results[0].annotationResults[0].safeSearchAnnotations;
-      console.log('Safe search results:');
-      safeSearchResults.forEach((result) => {
+      const explicitContentResults = results[0].annotationResults[0].safeSearchAnnotations;
+      console.log('Explicit content results:');
+      explicitContentResults.forEach((result) => {
         console.log(`Time: ${result.timeOffset / 1e6}s`);
         console.log(`\tAdult: ${likelihoods[result.adult]}`);
```

Comment:
Spoof, medical etc are removed (the API has not been returning non-empty values for those), and only adult (renamed pornography_likelihood) remains.